### ISSUE TITLE
Import Error in Debug Mode VS Code (UPDATED)

### ISSUE THREAD

**Issue by Johnathan-Graf** [rdkit/rdkit#4011]
I was running some code just last night and everything was working fine. I tried to run this morning and immediately I get:

Exception has occurred: ImportError
No module named 'rdkit'

YES, I HAVE THE CORRECT CONDA ENVIRONMENT ACTIVATED. YES, RDKIT HAS BEEN INSTALLED PROPERLY. I have been using this exact install/conda environment for about a month without any problems previously. I haven't changed anything.
Restarting my computer did not work.

**Configuration:**
 - RDKit version: 2017.09.1
 - OS: Windows 10
 - Python version: 3.6

**UPDATE:**
The error only occurs when running VS Code in debug mode. However, everything I previously said is still true. I was running the code in debug mode last night and it worked fine.

**Comment by greglandrum** [comment]
You don't provide enough info to be sure: how exactly are you launching the python instance which is trying to import the RDKit?

**Comment by Johnathan-Graf** [comment]
Launched Anaconda Navigator > Applications on... (dropbox, selected my 3.6 environment) > Launched VS Code

**Comment by greglandrum** [comment]
Apologies that I'm slow on the uptake here, but I'm afraid I still don't get it.

Did you build the rdkit yourself in debug mode and you're having problems launching it?

Or are you trying to run code which uses the rdkit in the debugger in VS Code?
If it's this second case: where did you install the rdkit from?

**Comment by Johnathan-Graf** [comment]
@greglandrum Sorry for the late response. No, I downloaded/installed rdkit with the instructions (https://www.rdkit.org/docs/Install.html). I was tried to run code that used rdkit in the debugger in VS Code.

I went back to my project and it is working properly now, so I guess its an issue on my pc? 

I have no idea why this happened, this is way out of my depth. However, I do insist that I had the correct conda environment activated - I quadruple checked that.
